www/www-deploy.yml

```diff
@@ -121,14 +119,14 @@
       dest: /bio-formats/downloads/
     - match: "~/site/products/ome-files-cpp/?$"
       dest: /ome-files
-    - match: "~/site/products/(?<link>.*)$"
```

Comment:
Similarly, how is this handled now? Also, why is `ome-files-cpp` still above?

Reply:
I can redirect it to /products if you like but this covers the partner products we don't have pages for any more.

Why would ome-files-cpp not be here? We have a specific page to redirect that to.